website/static/js/anonymousLogActionsList.json

```diff
@@ -92,7 +92,6 @@
     "subjects_updated": "A user updated the subjects",
     "view_only_link_added": "A user created a view-only link to a project",
     "view_only_link_removed": "A user removed a view-only link to a project",
-    "migrated_quickfiles": "QuickFiles were migrated into a public project",
```

Comment:
Don't these logs still exist in the db?

Reply:
Yeah you're right, this should cause problems, probably didn't specifically check for these logs. Note to self.